devnet/devnet.go

```diff
@@ -107,8 +112,9 @@ func (devnet *DevNet) IsAlive() bool {
 }
 
 type MintResponse struct {
-	NewBalance *big.Int `json:"new_balance"`
-	Unit       string   `json:"unit"`
+	NewBalance      string `json:"new_balance"`
```

Comment:
The new devnet version returns a string instead a number, and a new field. The compiler was unable to convert to bigInt